.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
```

Comment:
Can you make this section specific for Python source files? Also, this project contains a bit of c source - can you add a section for that, too? A Makefile section would be nice, too. My preference is to not apply these settings globally, since we do have different _types_ of source in this repo.

Reply:
Done. C sources seem to have the same conventions - both generated with cython and third-party libs.